concepts/enums/about.md

```diff
@@ -34,47 +32,47 @@ The `Direction` enum declared above is a simple enum with four variants: `North`
     }
 ```
 
-It is evident from the second example that each variant in the `Direction` enum has an associated value of `u128`. This kind of enum is instantiated as follows:
+It is evident from this example that each variant in the `Direction` enum has an associated value of `u128`. This kind of enum is instantiated as follows:
 
 ```Rust
     let direction = Direction::North(10);
 ```
 
-### Complex example of an enum with combined custom types
+## Complex example of an enum combined with custom types
+
+An enum can be declared with a combination of custom data types as its variants. In the example below, `Action` is the complex enum that as a collection of `Quit`, `Direction`, `SendMessage`, `ChangeAvatar`, and `ProfileState`.
 
 ```Rust
-    #[derive(Drop, Serde, Copy, starknet::Store)]
+    #[derive(Drop)]
     struct Move {
         up: u32,
         down: u32,
         right: u32,
         left: u32,
     }
 
-    #[derive(Drop, Serde, Copy, starknet::Store)]
+    #[derive(Drop)]
     enum UserCommand {
         Login,
         UpdateProfile,
         Logout,
     }
 
-    #[derive(Drop, Serde, Copy, starknet::Store)]
+    #[derive(Drop)]
     enum Action {
-        Quit,
-        Direction: Move,
-        SendMessage: felt252,
-        ChangeAvatarColor: (u8, u8, u8),
-        ProfileState: UserCommand
+        Quit, //  variant with no associated value
+        Direction: Move, // variant with the `Move` struct associated value
+        SendMessage: felt252, // a single felt252 variant
+        ChangeAvatarColor: (u8, u8, u8), // variant with the tuple of three associated value
+        ProfileState: UserCommand // variant with an enum associated value
     }
 ```
 
-An enum can be declared with a combination of custom data types as its variants. In the example above, the `Action` is the complex enum that as a collection of `Quit` (variant with no associated value), `Direction` (variant with the `Move` struct associated value), `SendMessage` (a single felt252 variant), `ChangeAvatar` (variant with the tuple of three associated value), and `ProfileState` (variant with an enum associated value).
-
-You can declare an enum inside and outside of a contract. If declared outside, then it should be imported inside a contract with the `use` keyword.
+You can declare an enum inside and outside of a Cairo file. If declared outside, then it should be imported inside the base file with the `use` keyword.
```

Comment:
An enum must be declared in a Cairo (_*.cairo_) file, what did you mean to say here exactly?

Reply:
I was going to explain how enums can be used, whether within a cairo file, or if in a very large codebase, it could be declared outside, as an util, then imported in the base file(.cairo).

Let me know if this information is not very much necessary.

Reply:
I would rather omit this completely, students will learn how to group code in modules and how to import them in the [modules concept](https://github.com/exercism/cairo/issues/38).